components/stripe/actions/create-refund/create-refund.mjs

```diff
@@ -33,22 +32,27 @@ export default {
       ],
     },
     reason: {
-      propDefinition: [
-        app,
-        "refund_reason",
+      type: "string",
+      label: "Reason",
+      description: "String indicating the reason for the refund. If set, possible values are duplicate, fraudulent, and requested_by_customer. If you believe the charge to be fraudulent, specifying fraudulent as the reason will add the associated card and email to your [block lists](https://docs.stripe.com/radar/lists), and will also help us improve our fraud detection algorithms.",
```

Comment:
```suggestion
      description: "String indicating the reason for the refund. If you believe the charge to be fraudulent, specifying fraudulent as the reason will add the associated card and email to your [block lists](https://docs.stripe.com/radar/lists), and will also help us improve our fraud detection algorithms.",
```
I think listing the options in the description is redundant if we already have the options prop

Reply:
I would also avoid using words such as "help us improve our fraud detection algorithms" since this is description is on Pipedream's, not the app's end.